### PR TITLE
BaSP-TG-67: Super Admin Form.

### PR DESCRIPTION
**Pull request by Gearyandres94**
Update
Save

### CODE REVIEW

**Review by HernanCazeneuve — CHANGES_REQUESTED** [review]
Hell of a job

File: src/Components/SuperAdmin/index.js [hunk 1]
```diff
@@ -1,30 +1,44 @@
+/* eslint-disable no-unused-vars */
```
Comment (CarVigil) [inline src/Components/SuperAdmin/index.js:1]: Remove this comment

File: src/Components/SuperAdmin/index.js [hunk 2]
```diff
@@ -62,41 +76,71 @@ function SuperAdminForm() {
...
+        <Input
+          name={'Password'}
+          type={'password'}
+          placeholder={'Enter the Password'}
+          onChange={setPassword}
+          value={password}
+        />
+        <div className={styles.superAdminDiv}>
+          <Button text={buttonText} handler={handleSubmit} />
+          <Button text={'Return'} handler={routeChange} />
```
Comment (CarVigil) [inline src/Components/SuperAdmin/index.js:143]: In all others forms, Return button is on the left.

File: src/Components/Shared/Input/index.js [hunk 3]
```diff
@@ -21,3 +21,5 @@ function Input({ name, type, value, placeholder, error, onChange }) {
   );
 }
 export default Input;
+
+//
```
Comment (CarVigil) [inline src/Components/Shared/Input/index.js:25]: There's an empty comment. Should be removed.

**Review by CarVigil — CHANGES_REQUESTED** [review]

**Review by CarVigil — DISMISSED** [review]
Excellent 👌

**Review by HernanCazeneuve — APPROVED** [review]

**Review by vadalamati — APPROVED** [review]
Good job!!!

**Review by HernanCazeneuve — APPROVED** [review]

**Review by FiolAxel — APPROVED** [review]

**Pull request merged**